pep-0678.rst

```diff
@@ -153,27 +161,30 @@ System-defined or "dunder" names (following the pattern ``__*__``) are part of
 the language specification, with `unassigned names reserved for future use and
 subject to breakage without warning
 <https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers>`__.
+We are also unaware of any code which *would* be broken by adding ``__notes__``.
 
-We are also unaware of any code which *would* be broken by adding ``__note__``;
-assigning to a ``.__note__`` attribute already *works* on current versions of
-Python - the note just won't be displayed with the traceback and exception
-message.
-
+While ``add_note()`` is a fairly common method name, our Google and GitHub
+searches did not find any exception classes which define such a method.
```

Comment:
It might be helpful to link to them. grep.app is a particularly useful tool here